packages/dart/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [3.1.14](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-3.1.13...dart-3.1.14) (2023-02-25)
```

Comment:
If we upgrade to dio 5, isn't this a breaking change for this SDK? The Parse Flutter SDK references this `^3.1.2` then there will be a conflict. So we need to release the dart package as `4.0.0`, right?

Reply:
No, I don't think it is a breaking change because we are only updating the packages and there are no changes at the api level.
I don't think package conflict is a breaking change. Because you can still use the old package.

Reply:
Let's assume an app has the dependency `flutter: ^3.1.3`. That flutter version has the dependency `parse_server_sdk: ^3.1.2`. What happens on the next `flutter pub get`? If I understand it correctly, it will pull `flutter: 3.1.3` with dio 4 and that will pull the latest non-breaking release of `parse_server_sdk: 3.1.14` with dio 5. Is that correct?

Reply:
Yes, it happens in the same way and there is no conflict

Reply:
Shouldn't there be a conflict if the flutter SDK uses dio 4 but its dependency dart SDK uses dio 5?

Reply:
> Shouldn't there be a conflict if the flutter package uses dio 4 but its dependency the dart package uses dio 5?

Yes, in this case we have a conflict

Reply:
I don't see a way around this conflict. If we release the dart package as `3.x` then there's the conflict, so I guess releasing as 4.x is the only way around this, right?

Reply:
Yes, it is true, I think it would be better to upgrade to version ```4.0.0```

Reply:
Alright, thanks for confirming